tests/network/vmi_istio.go

```diff
@@ -56,17 +56,22 @@ const (
 
 var _ = SIGDescribe("[Serial] Istio", func() {
 	var (
-		err        error
-		vmi        *v1.VirtualMachineInstance
-		virtClient kubecli.KubevirtClient
+		err           error
+		vmi           *v1.VirtualMachineInstance
+		virtClient    kubecli.KubevirtClient
+		vmiPorts      []v1.Port
+		explicitPorts = []v1.Port{
```

Comment:
I should have asked it before, but can you please add a comment explaining why the fact that the port is declared or not declared in the service is important? It's pretty difficult to understand, especially because the service in this test file is just created but never used.

Reply:
Added comment